doc/getting_started/index.rst

```diff
@@ -22,18 +18,13 @@ you'll:
 Select and Update OS
 ********************
 
-Zephyr development depends on an up-to-date host system and common build system
-tools. First, make sure your development system OS is updated:
+Click the operating system you are using.
```

Comment:
When looking at the rendered doc, I think there should be just a single line explaining why users should do those commands, so maybe add a little:
```suggestion
Click the operating system you are using and follow the guide to ensure it is up-to-date.
```

Reply:
I think this is implied by the title "Select and Update OS"

Reply:
yeah, but when I first read it, I just read the content and my eyes didn't notice the title.
Not sure if i'm the only one :laughing:

Reply:
I guess adding more text isn't likely to make eyes glaze less, though. That's exactly why I'm trying to strip this down and remove any text I can -- people generally ignore text and just start copy pasting command lines into their terminals as fast as they can, in my experience.